test-scripts/ballerina-test-automation/src/test/java/io/ballerina/test/UpdateDistTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.ballerina.test;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+
+public class UpdateDistTest {
+    String version = "1.2.0";
+    String specVersion = "2020R1";
+    String toolVersion = "0.8.5";
```

Comment:
Can we parameterize these

Reply:
+1 
Included them in gradle.properties file